boxsdk/object/base_api_json_object.py

```diff
@@ -37,28 +41,13 @@ def __getitem__(self, item):
 
     def __repr__(self):
         """Base class override. Return a human-readable representation using the Box ID or name of the object."""
-        description = '<Box {0} - {1}>'.format(self.__class__.__name__, self._description)
+        extra_description = ' - {0}'.format(self._description) if self._description else ''
+        description = '<Box {0}{1}'.format(self.__class__.__name__, extra_description)
         if six.PY2:
             return description.encode('utf-8')
         else:
             return description
 
     @property
     def _description(self):
```

Comment:
Since this is now a protected (instead of private) method that we'll be overriding in subclasses, we should add a docstring.